### DISCUSSION TITLE
In an M2O field pointing to an UUID, with a template on "name", we should not be able to edit

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16785] [Feature Requests]
right in the input to force to click on the radio list.

Parent <= M2O => Child

Parent 
  id
  childfield with template on Chlld.name

Child
  id
  name

When inserting a Parent, we can type directly in the childfield input.
This is not relevant when the link is on an UUID.
The field should not allow to type, or open the Child list when we begin typing in the childfield input/

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
